webapp/src/components/HomePage/Sidebar/ExpandedSidebar/WalletContributions/ContributionRow/ContributionRow.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import DistrictLink from '../DistrictLink'
+
+class ContributionRow extends React.PureComponent {
+  render() {
+    const { contribution } = this.props
+
+    return (
+      <div className="table-row">
+        <div className="col col-district">
+          <DistrictLink contribution={contribution} />
+        </div>
+        <div className="col col-contributed">
+          {contribution.land_count.toLocaleString()} LAND ({contribution.land_count *
+            1000}{' '}
```

Comment:
What do think about adding an object to handle this type of conversions? Maybe we could have a `land` domain similar to what we had here: https://github.com/decentraland/auction/blob/master/webapp/src/lib/land.js

so we can do `land.convert(contribution.land_count, 'mana')`